server/routers/transactions.router.js

```diff
@@ -0,0 +1,20 @@
+import { Router } from 'express';
+
+const transactionRouter = Router();
+
+transactionRouter.get('/' /*to do auth, getTransactions*/);
+
+transactionRouter.post('/' /*to do auth, createTransaction*/);
+
+transactionRouter.get('/:id' /*to do auth, getTransactionById*/);
```

Comment:
getOwnerTransactionsById

Reply:
jw :) Napisałam, że to propozycje nazw :)